src/__integrationtests__/Balance.spec.ts

```diff
@@ -20,7 +25,7 @@ import {
   wannabeFaucet,
 } from './utils'
 
-let blockchain: IBlockchainApi | undefined
```

Comment:
I hate `undefined`

Reply:
actually that is a change that is no longer necessary... should I revert?

Reply:
So, you will need undefined.

Reply:
the original is indeed the better description of what's going on, if that's what you mean